### DISCUSSION TITLE
Compatibility range is not a range?

### DISCUSSION THREAD

**Discussion by matejvasek** [spring-io/start.spring.io#1394] [Q&A] [answered]
Here the last `compatibilityRange` is not really a range but a singular value, why?
https://github.com/spring-io/start.spring.io/blob/main/start-site/src/main/resources/application.yml#L83-L90

**Comment by wilkinsona** [comment, marked as answer]
It's modelled on [Maven's version range syntax](https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html). A single value means greater than or equal to. This is also [described](https://docs.spring.io/initializr/docs/current/reference/html/#dependencies-compatibility-range) in the documentation.

**Comment by matejvasek** [comment]
Thank you!
